src/components/custom-field-input-date/custom-field-input-date.jsx

```diff
@@ -216,11 +231,11 @@ const CustomFieldInputDate = forwardRef(function CustomFieldInputDate(props, ref
         <AbstractCustomField
           {...sharedProps}
           defaultValue={currentValue}
-          errorText={errorText()}
+          errorText={isValid ? '' : errorText()}
           id={props.id}
           key={`${props.id}-editing`}
-          min={formatDateString(props.min)}
-          max={formatDateString(props.max)}
+          min={formatDateString(apiLimits.min)}
```

Comment:
Instead of ensuring min/max all the time, I think it may be best to allow setting a min/max via props and then set the default value to be the limit instead. That way we can still use this component as a date picker with min/max values that an engineer can set, or the default one for good data.

Reply:
We were going to ask about this since this is a custom fields date picker (and will always have to conform to the good data api) and not a generic date picker. We went with this way for now but definitely were considering just validating on the min max props

Reply:
It definitely feels odd to implement it with such a tight integration to an API that we don't own, but if we need to do this for every `CustomFieldInputDate` anyway, this might be better.

Reply:
Perhaps the "meet in the middle" is to move it to props, and have the default value for those props be what the gooddata API expects? 🤔

Reply:
Im down with that. It seems like these custom fields components have an identity crisis of being generic and specific to mavenlink custom fields. Do you all think they are still a good fit for MDS?

Reply:
@Bapark They definitely aren't generic, and are only for Custom Fields. That being said, I suppose there is an identity crisis, but more on the side of "Custom Fields API vs Gooddata API/integration" IMHO.

Reply:
@Bloodyaugust The Bigmaven custom field API actually throws errors if the provided date is out of this date range (the bigmaven API itself is limited due to the good data limitation) I guess what I mean by an 'identity crisis' is with this component, is the consumer of this component managing state validity or is this component managing itself? I will hit you guys up in slack and we can think about it to arrive in a good spot.

Reply:
We will end up extracting a generic MDS Date from CustomFieldInputDate in the near future.

We can hard-code the min/max in CustomFieldInputDate but leave MDS Date with the generic props interface. 

I wonder if there is a pattern for bigmaven date validations. Or maybe this brings up a bigger question whether bigmaven should have consistent min/max values for date attributes.